### DISCUSSION TITLE
Can In-App messaging triggered by Audience trigger ?

### DISCUSSION THREAD

**Discussion by fumito-ito** [firebase/firebase-ios-sdk#8756] [Q&A] [answered]
Hi, my project uses firebase in-app messaging and firebase audience. I want to show message to updated users by in-app messaging. I try following steps but any message appears. Does In-App messaging support audience trigger ?

- create `Updated Users` audience in Firebase Audience. In analytics console, it was detected correctly .
- add Audience trigger named `update_from_older_version` to `Updated Users` audinece
- add schedule trigger  `update_from_older_version` to firebase in-app messaging campaign

thanks.

**Comment by paulb777** [comment, marked as answer]
I created Issue #8808 to track.
